index.js

```diff
@@ -307,5 +307,93 @@ class Stack {
   }
 }
 
+class SinglyLinkedList {
```

Comment:
Please create a single class for Singly as well as Doubly Linked List. We will add configurations to the LinkedList Class while instantiation so as to reduce redundancy of code.